RELEASE/scripts/sl_ascend.ash

```diff
@@ -12816,6 +12857,12 @@ boolean L9_chasmBuild()
 			slEquip(it);
 		}
 
+		//Remove Ur-kel's if we have it running
```

Comment:
You added a section in `postsool` that removes Ur-kel's, so I think this is redundant.

Reply:
Ah, yes, removed.